injection/payload.html

```diff
@@ -0,0 +1,72 @@
+<script language="JavaScript">

```

Comment:
The injection payload should not be committed to the repository. It's part of the configuration, not part of the program.